.vscode/launch.json

```diff
@@ -1,20 +1,111 @@
 {
-    "version": "0.2.0",
+   "version": "0.2.0",
     "configurations": [
+        {
+            "name": "(gdb) Launch Serac",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build-gcc@7.4.0-debug/bin/serac",
+            "args": ["-m", "${workspaceFolder}/data/beam-hex.mesh"],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}/build-gcc@7.4.0-debug/bin/",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                }
+            ]
+        },
+        {
+            "name": "(gdb) mpi Dynamic Test",
```

Comment:
These need to get changed or removed, since they're specific to my paths and compiler choice. For some reason, with the mpi ones, the variable for the cmake build directory didn't expand correctly (missing slashes) for the prefix to the binary, so I had to put the entire path to get things working.